diff --git a/node_modules/ermis-chat-js-sdk/dist/browser.js b/node_modules/ermis-chat-js-sdk/dist/browser.js
index a892fe7..1c63f76 100644
--- a/node_modules/ermis-chat-js-sdk/dist/browser.js
+++ b/node_modules/ermis-chat-js-sdk/dist/browser.js
@@ -5175,7 +5175,7 @@ var StableWSConnection = /*#__PURE__*/function () {
 
       var token = _this.client.tokenManager.getToken();
 
-      return "".concat(_this.client.wsBaseURL, "/connect?json=").concat(qs, "&api_key=").concat(_this.client.key, "&authorization=").concat(token, "&stream-auth-type=").concat(_this.client.getAuthType(), "&X-Stream-Client=").concat(_this.client.getUserAgent());
+      return encodeURI("".concat(_this.client.wsBaseURL, "/connect?json=").concat(qs, "&api_key=").concat(_this.client.key, "&authorization=").concat(token, "&stream-auth-type=").concat(_this.client.getAuthType(), "&X-Stream-Client=").concat(_this.client.getUserAgent()));
     });
 
     _defineProperty__default['default'](this, "onlineStatusChanged", function (event) {
